src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -725,13 +725,13 @@ public static XmlEmptyElementSyntax XmlThreadSafetyElement()
         /// <summary>
         /// Creates a threadsafty element within an xml documentation comment.
```

Comment:
This may be a first for me ... a PR which reduces the number of typos in the code base.